integration/query_comparision_test.go

```diff
@@ -1074,6 +1074,190 @@ func TestQueryComparisonNin(t *testing.T) {
 	}
 }
 
-// $in
+func TestQueryComparisonNe(t *testing.T) {
+	t.Parallel()
+	providers := []shareddata.Provider{shareddata.Scalars, shareddata.Composites}
+	ctx, collection := setup(t, providers...)
+
+	for name, tc := range map[string]struct {
+		value         any
+		valueToBeLost string
```

Comment:
That's ID, not a value. A better name might be `unexpectedID`

Reply:
fixed